### PR TITLE
add instructions for resetting the database

### PR DESCRIPTION
**Pull request by benadida**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,22 @@ Then, to create an administrator for the organization:
 
 which returns the `user_id`.
 
+### Resetting the Database When Upgrading Arlo
+
+If you're upgrading Arlo, right now the only way is to destroy and
+recreate the database. The easiest way to do that, if your database
+connection is set up properly, is:
+
+`pipenv run python resetdb.py`
```
Comment (jonahkagan) [inline README.md]: Not `make resetdb`?

File: README.md [hunk 2]
```diff
@@ -120,6 +120,22 @@ Then, to create an administrator for the organization:
...
+
+If you're running on Heroku or on another system where database
+destruction and recreation cannot be done from Python, because the
+database is provisioned externally, the steps are then:
+
+- clear the database, e.g. on heroku `heroku pg:reset -a <app_name>`
+- create just the data model, e.g. on heroku `heroku run -a <app_name> python resetdb.py --skip-db-creation`
+- you may need to restart serveres, e.g. on heroku `heroku restart -a <app_name>`
```
Comment (jonahkagan) [inline README.md]:
```suggestion
- you may need to restart servers, e.g. on heroku `heroku restart -a <app_name>`
```

File: README.md [hunk 3]
```diff
@@ -120,6 +120,22 @@ Then, to create an administrator for the organization:
 
 which returns the `user_id`.
 
+### Resetting the Database When Upgrading Arlo
+
+If you're upgrading Arlo, right now the only way is to destroy and
```
Comment (jonahkagan) [inline README.md]: What exactly does upgrading mean? Does it mean moving between versions where the database schema changed? Are these instructions standing in place of a strategy for db migrations?

Comment (benadida) [reply README.md]: exactly, we don't have db migrations yet and we've explicitly chosen not to have them until we really need them. Which might be soon :)

File: README.md (hunk 1 above)
Comment (benadida) [reply README.md]: 👍

**Pull request merged**